src/data/real/basic.lean

```diff
@@ -625,19 +625,18 @@ begin
     apply sub_le_sub_left,
     cases le_total 0 y with hy hy,
     { rw mul_self_sqrt hy,
-      exact calc
-        y ≤ 2 * y - y : by ring
-        ... = 2 * (sqrt y * sqrt y) - y : by rw mul_self_sqrt hy
-        ... ≤ 2 * (sqrt x * sqrt y) - y :
-        begin
-          apply sub_le_sub_right,
-          apply mul_le_mul_of_nonneg_left,
-          apply mul_le_mul_of_nonneg_right,
-          exact sqrt_le_sqrt h,
-          exact sqrt_nonneg _,
-          norm_num
-        end
-        ... ≤  2 * sqrt x * sqrt y - y : by ring },
+      calc y ≤ 2 * y - y : by ring
+         ... = 2 * (sqrt y * sqrt y) - y : by rw mul_self_sqrt hy
+         ... ≤ 2 * (sqrt x * sqrt y) - y :
+         begin
+           apply sub_le_sub_right,
```

Comment:
It seems like that in these proof block you didn't use `{}`, and I guess that at times there will be more than 1 goal. Please restructure it a bit. (You could feed things like `sqrt_nonneg _` directly to the lemma that needs it.)